### PR TITLE
Adding script for cleaning up old data

### PR DESCRIPTION
**Pull request by halkyon**

### CODE REVIEW

**Comment by stojg** [comment]
Can I have a dry-run option that lists what will be deleted so I wouldn't be so scared to run this task?

**Comment by halkyon** [comment]
@stojg yup, good idea

**Comment by halkyon** [comment]
@stojg `--dry-run` arg added

File: code/tasks/CleanupOrphanedRecordsTask.php [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+			$log('Running in dry-run mode. No data will be deleted');
+		}
+
+		$count = 0;
+
+		foreach (DNEnvironment::get() as $environment) {
+			$project = $environment->Project();
+			if (!$project || !$project->exists()) {
+				$log(sprintf(
+					'Environment (ID %s, Created: %s) is linked to a non-existent project. Deleting',
```
Comment (stojg) [inline code/tasks/CleanupOrphanedRecordsTask.php]: Can you add the name of the environment to the output please?
`... is linked to a non-existent project 'Banana'`

**Pull request merged**